R/sanity_model.R

```diff
@@ -131,9 +132,8 @@ sanitize_model <- function(model,
                            vcov = NULL,
                            ...) {
 
-    # tidymodels appear to store the model fit in `model[["fit"]]`
-    if (inherits(model, "model_fit") && "fit" %in% names(model)) {
```

Comment:
parsnip models have class `model_spec` until they are fitted, at which point they become `model_fit`s and always gain that `"fit"` slot.